includes/blast_ui.admin.inc

```diff
@@ -156,6 +164,100 @@ function blast_ui_admin_form_validate($form, &$form_state) {
 
 }
 
+
+/*
+**
+** custom method disabled the web services 
+**
+*/
```

Comment:
Please follow Drupal coding standards as described here: https://www.drupal.org/docs/develop/standards/coding-standards. Most importantly follow the standards for function documention blocks: https://www.drupal.org/docs/develop/coding-standards/api-documentation-and-comment-standards including the placement and spacing of the asterisks (*) since this impacts the quality of automatically generated documentation. For example,
```
/**
 * Disables the web services.
 * This function disables the Drupal service module as well as removing the service endpoint
 * for Blast services.
 */
```
Also, there is no need to call it a custom function/method.
NOTE: I'm not going to comment on all the function doc blocks that don't match standards but would appreciate all of them fixed. Thanks!